src/_includes/graphql/customer-output-24.md

```diff
@@ -14,9 +25,8 @@ Attribute |  Data Type | Description
 `is_subscribed` | Boolean | Indicates whether the customer is subscribed to the company's newsletter
 `lastname` | String | The customer's family name
 `middlename` |String | The customer's middle name
+`orders(<FilterCriteria>)` | {{ customeroutput_text }} | Defines a filter for returning previous customer orders{{ customeraddress_text }}
```

Comment:
I think the description for this should be more along the lines of "A list of the customer's placed orders"